app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -191,6 +193,10 @@ snippet: true
     done
   <% end %>
 
+  <% if host_param_true?('subscription_manager_auto_attach', false) -%>
```

Comment:
One thing to keep in mind here is the interaction with Activation Keys.

An activation key may specify that auto-attach is enabled, so auto-attach my have already ran if registered with such a key. Or it was specifically **disabled** on the activation key. So, running it here acts as an override of the preference set on the activation key. I don't see any immediate issue with that since this parameter will be intentionally set and that is probably the desired effect.